examples/chat/README.md

```diff
@@ -40,28 +40,28 @@ This will emit a trace line that holds the information needed to:
 - join the chat room (with `--room_id`).
 
 ```log
-2019-10-24 10:47:20  INFO  chat.cc : 242 : Join this room with --address=127.0.0.1:32889 --room_id=NKsceNlg69UbcvryfzmFGnMv9qnZ0DYh6u6gJxujnPPxvHsxMehoD368sumKawVaq9WaSkzrcStoNYLvVNdzhA==
+2019-10-24 10:47:20  INFO  chat.cc : 242 : Join this room with --address=localhost:32889 --room_id=NKsceNlg69UbcvryfzmFGnMv9qnZ0DYh6u6gJxujnPPxvHsxMehoD368sumKawVaq9WaSkzrcStoNYLvVNdzhA==
```

Comment:
(pre-existing) Should the port be :8080 in all the sample log output now?

Reply:
Done